telemetry/gparser/parser.py

```diff
@@ -250,17 +255,26 @@ def get_payload_raw(self):
         return payload
     
     def get_payload_parsed(self):
-        payload = []
-        payload_split = {"procedure":[], "param":[]}
-        for payload in self.payload_raw:
-            procedure_param = payload.split("[")
-            payload_split["procedure"].append(procedure_param[0])
+        numPayload = len(self.payload_raw)
+        procedure = list(range(numPayload))
```

Comment:
Can numPayload be formatted in snake case ex: num_payload

Reply:
https://github.com/sdgtt/telemetry/pull/26/commits/4cfd03297d9aa3171af7baec39119b0c169af80c